_inc/client/plans/single-product-backup.jsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { AccentedCard } from './accented-card';
+import Gridicon from 'components/gridicon';
+
+export function SingleProductBackup( props ) {
+	const { sitePlans } = props;
+
+	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
+
+	return (
+		<>
+			<h1 className="plans-section__header">Single Products</h1>
+			<h2 className="plans-section__subheader">Just looking for backups? We've got you covered.</h2>
+			<div style={ { display: 'flex', justifyContent: 'center', marginBottom: '10px' } }>
+				<AccentedCard>
+					{ {
+						header: <SingleProductBackupHeader sitePlans={ sitePlans } />,
+						body: (
+							<SingleProductBackupBody
+								selectedBackupType={ selectedBackupType }
+								setSelectedBackupType={ setSelectedBackupType }
+								sitePlans={ sitePlans }
+							/>
+						),
+					} }
+				</AccentedCard>
+			</div>
+		</>
+	);
+}
+
+function SingleProductBackupHeader( props ) {
+	const { sitePlans } = props;
+
+	return (
+		<div
+			style={ {
+				display: 'flex',
+				flexDirection: 'row',
+				justifyContent: 'space-between',
+				alignContent: 'center',
+			} }
+		>
+			<h3>{ __( 'Jetpack Backup' ) }</h3>
+			{ sitePlans && (
+				<PlanPriceDisplay
+					dailyPrice={ sitePlans[ 'daily-backup' ].price.yearly.text }
+					yearlyPrice={ sitePlans[ 'realtime-backup' ].price.yearly.amount }
+				/>
+			) }
+		</div>
+	);
+}
+
+export function PlanPriceDisplay( props ) {
+	const { dailyPrice, yearlyPrice } = props;
+	const perYearPriceRange = `${ dailyPrice }-${ yearlyPrice } /year`;
+
+	return (
+		<div
+			style={ {
+				display: 'flex',
+				flexDirection: 'row',
+				justifyContent: 'space-between',
+				alignContent: 'center',
+			} }
+		>
+			<SlashedPrice />
+			<div className="plans-price__container">
+				<span className="plans-price__span">{ perYearPriceRange }</span>
+			</div>
+		</div>
+	);
+}
+
+function SlashedPrice() {
```

Comment:
Agreed, but I'd rather do that in another PR. Making components shared deserves its own review since I may not have chosen the best API for some of these.

Reply:
Totally! But it also makes sense to prepare and merge these PRs before this one, to avoid polluting the history with unnecessary changes. Does that work for you?